python/vesin-torch/tests/test_autograd.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import torch
+
+from vesin.torch import NeighborList
+
+
+@pytest.mark.parametrize("full_list", [True, False])
```

Comment:
I don't know how useful but one could add a device parametrization. I did something here

https://github.com/lab-cosmo/torch-pme/blob/c45015bd219c9f5dba818476dc9f0a6bb5bb20d8/tests/calculators/test_workflow.py#L15-L17

This will of course not run on CI but if you run the test on a machine that has an accelerator.

Reply:
I think the code will error out if you try to run it on something else than "cpu". I'll add a test about that!

The long term idea is to have actual implementation for other devices, see https://github.com/Luthaf/vesin/blob/eb01ac3f06484c32d714674c79c1c1dc131471cf/include/vesin.h#L50-L57 which can be extended to add CUDA, MPS, ... down the line.

Reply:
I see, yeah maybe checking the error is useful.